package.json

```diff
@@ -146,6 +146,7 @@
     "webpack": "4.44.1"
   },
   "resolutions": {
-    "@types/vfile-message": "1.0.1"
+    "@types/vfile-message": "1.0.1",
+    "**/jest-haste-map/fsevents": "2.2.1"
```

Comment:
This this comment worth considering? https://github.com/facebook/jest/issues/10777#issuecomment-722299866

Reply:
I tried to ensure that the new patch still make it through to support linux/windows, but I guess that didn't work. Let me give that a shot.

Reply:
This should be fixed now. Had to mess with the lockfile a little more than the comments suggested.

Reply:
Even better. :pray: